components/script/dom/webidls/CustomElementRegistry.webidl

```diff
@@ -6,13 +6,15 @@
 [Pref="dom.customelements.enabled"]
 interface CustomElementRegistry {
   [Throws, CEReactions]
-  void define(DOMString name, Function constructor_, optional ElementDefinitionOptions options);
+  void define(DOMString name, CustomElementConstructor constructor_, optional ElementDefinitionOptions options);
 
   any get(DOMString name);
 
   Promise<void> whenDefined(DOMString name);
 };
 
+callback CustomElementConstructor = any(any... arguments);
```

Comment:
Why does this include `any... arguments`?

Reply:
`any()` means that it is a function that can return any value but accepts no arguments.